README.md

```diff
@@ -20,6 +20,7 @@ Conference | Dates | Proposals | Location | Twitter
 Conference | Dates | Proposals | Location | Twitter
 -----------|-------|-----------|----------|---------
 [CC Congress](https://events.ccc.de) | Dec 27th - 30th | Unknown | Unknown, Germany | [![](https://user-images.githubusercontent.com/10602/44954380-6a3ab480-aea1-11e8-8ce7-ad540063f4de.png)](https://twitter.com/ccc)
+[JSConf Budapest](https://jsconfbp.com) | Sept 26th - 27th | 2018 Dec 30 | Budapest, Hungary | [![](https://user-images.githubusercontent.com/10602/44954380-6a3ab480-aea1-11e8-8ce7-ad540063f4de.png)](https://twitter.com/jsconfbp)
```

Comment:
I think it's implied that the CFP date is before the conference, so I'd rather leave out the year from the CFP column :)

```suggestion
[JSConf Budapest](https://jsconfbp.com) | Sept 26th - 27th | Dec 30th | Budapest, Hungary | [![](https://user-images.githubusercontent.com/10602/44954380-6a3ab480-aea1-11e8-8ce7-ad540063f4de.png)](https://twitter.com/jsconfbp)
```